### PR TITLE
Add FAQ entry reg. installation on iOS

### PR DESCRIPTION
**Pull request by molarya1**
This commit adds an FAQ entry reg. the requirements for installing the app on an iPhone.

This PR addresses the following issue: https://github.com/corona-warn-app/cwa-website/issues/593

---
Internal Tracking ID: [EXPOSUREAPP-12774](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-12774)

### CODE REVIEW

File: src/data/faq.json [hunk 1]
```diff
@@ -1848,6 +1848,19 @@
                         "id": "ios",
                         "indexed": true,
                         "accordion": [
+                            {
+                                "title": "[Apple/iOS]: What are the requirements for installing the Corona-Warn-App on an iPhone?",
+                                "anchor": "ios_install_requirements",
+                                "active": false,
+                                "textblock": [
+                                    "If you want to use the Corona-Warn App, at least <b>iOS [ios.minimum-required-os-version]</b> must be installed on your iPhone. Thus, the Corona-Warn App is only available on devices which support at least this iOS version. To check which iOS version you are using, please follow the procedure described here: <a href='https://support.apple.com/en-us/HT201685' target='_blank' rel='noopener noreferrer'>Find the software version on your iPhone, iPad, or iPod (Apple Support)</a>.",
```
Comment (MikeMcC399) [inline src/data/faq.json:1856]: Write "Corona-Warn-App" not "Corona-Warn App" when referring to the app. (Multiple places).

File: src/data/faq_de.json [hunk 2]
```diff
@@ -1847,6 +1847,18 @@
                         "id": "ios",
                         "indexed": true,
                         "accordion": [
+                            {
+                                "title": "[Apple/iOS]: Was sind die Voraussetzungen für die Installation der Corona-Warn-App auf einem iPhone?",
+                                "anchor": "ios_install_requirements",
+                                "textblock": [
+                                    "Wollen Sie die Corona-Warn App benutzen, muss mindestens <b>iOS [ios.minimum-required-os-version]</b> auf ihrem iPhone installiert sein. Somit ist die Corona-Warn App nur auf iPhones, die diese iOS-Version unterstützen, verfügbar. Um zu überprüfen welche iOS-Version Sie verwenden, gehen Sie bitte wie hier beschrieben vor: <a href='https://support.apple.com/de-de/HT201685' target='_blank' rel='noopener noreferrer'>Softwareversion auf dem iPhone, iPad oder iPod ermitteln (Apple Support)</a>.",
```
Comment (MikeMcC399) [inline src/data/faq_de.json:1854]: "auf ihrem iPhone" => "auf Ihrem iPhone"

File: src/data/faq_de.json [hunk 3]
```diff
@@ -1847,6 +1847,18 @@
...
+                                "anchor": "ios_install_requirements",
+                                "textblock": [
+                                    "Wollen Sie die Corona-Warn App benutzen, muss mindestens <b>iOS [ios.minimum-required-os-version]</b> auf ihrem iPhone installiert sein. Somit ist die Corona-Warn App nur auf iPhones, die diese iOS-Version unterstützen, verfügbar. Um zu überprüfen welche iOS-Version Sie verwenden, gehen Sie bitte wie hier beschrieben vor: <a href='https://support.apple.com/de-de/HT201685' target='_blank' rel='noopener noreferrer'>Softwareversion auf dem iPhone, iPad oder iPod ermitteln (Apple Support)</a>.",
+                                    "Sollten Sie <b>iOS [ios.minimum-required-os-version]</b> oder neuer installiert haben, könne Sie die App aus dem App Store installieren. Haben Sie eine ältere iOS-Version installiert, müssen Sie erst ein Softwareupdate durchführen. Um das zu tun, folgen Sie bitte diesen Schritten: <a href='https://support.apple.com/de-de/HT204204' target='_blank' rel='noopener noreferrer'>iPhone, iPad oder iPod touch aktualisieren (Apple Support)</a>.",
```
Comment (MikeMcC399) [inline src/data/faq_de.json:1855]: "könne Sie" => "können Sie"

File: src/data/faq_de.json [hunk 4]
```diff
@@ -1847,6 +1847,18 @@
...
+                                    "Siehe auch:",
+                                    "<a href='https://www.coronawarn.app/de/faq/results/#ios135' target='_blank' rel='noopener noreferrer'>Warum ist iOS 13.7 bzw. iOS 12.5 Mindestvoraussetzung?</a>.",
+                                    "<a href='https://www.coronawarn.app/de/faq/results/#minimum_requirements' target='_blank' rel='noopener noreferrer'>Welche Systemvoraussetzungen muss mein Smartphone erfüllen?</a>."
```
Comment (MikeMcC399) [inline src/data/faq_de.json:1859]: Convert links to relative

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
- Always write the app as "Corona-Warn-App" in English and German.
- For internal links, always make these relative. Do not use absolute URLs in this case.

File: src/data/faq.json (hunk 1 above)
Comment (Ein-Tim) [reply src/data/faq.json:1856]:
```suggestion
                                    "If you want to use the Corona-Warn-App, at least <b>iOS [ios.minimum-required-os-version]</b> must be installed on your iPhone. Thus, the Corona-Warn-App is only available on devices which support at least this iOS version. To check which iOS version you are using, please follow the procedure described here: <a href='https://support.apple.com/en-us/HT201685' target='_blank' rel='noopener noreferrer'>Find the software version on your iPhone, iPad, or iPod (Apple Support)</a>.",
```

File: src/data/faq_de.json (hunk 3 above)
Comment (Ein-Tim) [reply src/data/faq_de.json:1855]:
```suggestion
                                    "Sollten Sie <b>iOS [ios.minimum-required-os-version]</b> oder neuer installiert haben, können Sie die App aus dem App Store installieren. Haben Sie eine ältere iOS-Version installiert, müssen Sie erst ein Softwareupdate durchführen. Um das zu tun, folgen Sie bitte diesen Schritten: <a href='https://support.apple.com/de-de/HT204204' target='_blank' rel='noopener noreferrer'>iPhone, iPad oder iPod touch aktualisieren (Apple Support)</a>.",
```

**Comment by Ein-Tim** [comment]
@MikeMcC399 
Thanks for the review, I will address these issues soon.

File: src/data/faq.json [hunk 5]
```diff
@@ -1848,6 +1848,19 @@
...
+                                    "Since the Corona-Warn App receives regular updates which contain new functions but also bugfixes, you should always use the latest version of the App. The current version of the Corona-Warn App is <b>[ios.current-app-version]</b>. To check which version of the Corona-Warn App you are using, open the App, scroll down to the bottom and click on <q>App Information</q>. There you will see the currently installed version. If this is not the latest version, please follow these steps to update the app: <a href='https://support.apple.com/en-us/HT202180' target='_blank' rel='noopener noreferrer'>How to manually update apps on your Apple device (Apple Support)</a>.",
+                                    "See also:",
+                                    "<a href='https://www.coronawarn.app/en/faq/results/#ios135' target='_blank' rel='noopener noreferrer'>Why is iOS 13.7 or iOS 12.5, respectively, a minimum requirement?</a>",
+                                    "<a href='https://www.coronawarn.app/en/faq/results/#minimum_requirements' target='_blank' rel='noopener noreferrer'>What are the minimum operating system requirements for my phone?</a>"
```
[the remark this answers is not in the document]
Comment (Ein-Tim) [reply src/data/faq.json]: Fixed with [`c8638dd`](https://github.com/corona-warn-app/cwa-website/pull/2738/commits/c8638ddf17e9100f98c8c61b62384ff00e69adcc).

File: src/data/faq.json (hunk 1 above)
Comment (Ein-Tim) [reply src/data/faq.json]: Fixed with [`c8638dd`](https://github.com/corona-warn-app/cwa-website/pull/2738/commits/c8638ddf17e9100f98c8c61b62384ff00e69adcc).

**Review by larswmh — APPROVED** [review]

**Review by dsarkar — APPROVED** [review]

**Comment by Ein-Tim** [comment]
**DO NOT MERGE** The replacements do not work!

**Comment by Ein-Tim** [comment]
Okay, I have never noticed this, but it seems like this is a general problem locally.

Try e.g. https://www.coronawarn.app/en/faq/results/#minimum_requirements. Access it locally (http://localhost:8000/en/faq/results/#minimum_requirements) and on the PROD server.
Locally, you will see this: 

Bildschirmfoto 2022-04-12 um 15 41 53

While on PROD you will see this: 

Bildschirmfoto 2022-04-12 um 15 42 01


So, this is not a problem with this PR but a general issue. I will open an issue later on. This RR can be safely merged.

**Comment by dsarkar** [comment]
@molarya1 @Ein-Tim @MikeMcC399 Thanks!

**Comment by Ein-Tim** [comment]
That solved it @dsarkar! Thanks!

**Pull request merged**